flowback/ledger/models.py

```diff
@@ -0,0 +1,33 @@
+from django.db import models
+from django.utils import timezone
+
+from flowback.user.models import User
+
+class Account(models.Model):
+    account_number = models.CharField(max_length=20)
+    account_name = models.CharField(max_length=100)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+
+    def balance(self):
+        debit_total = self.transactions.filter(debit_amount__isnull=False).aggregate(models.Sum('debit_amount'))['debit_amount__sum'] or 0
```

Comment:
use .get instead of getting the field value directly to avoid field not found response, aswell as not needing to use 'or 0'

replace ['debit_amount__sum'] with .get('debit_amount__sum', 0')

do the same with credit_total